scripts/centos/epel-7.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash -eux
-
-rpm -Uvh http://epel.osuosl.org/epel-release-latest-7.noarch.rpm
-
-# use our mirror
-sed -i -e 's/^\(mirrorlist.*\)/#\1/g' /etc/yum.repos.d/epel.repo
-sed -i -e 's/^#baseurl=.*pub\/epel\/7\/$basearch$/baseurl=http:\/\/epel.osuosl.org\/7\/$basearch\//' /etc/yum.repos.d/epel.repo
```

Comment:
This file is still used in `centos-7-ppc64-openstack.json` and `centos-7-ppc64le-openstack.json`.